src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -55,6 +60,78 @@ def __repr__(self):
 #                sql db                       #
 ###############################################
 
+
+class Engine(Enum):  # pylint: disable=too-few-public-methods
+    """SQL RDBMS engine type."""
+    db = 'db'
+    dw = 'dw'
+
+
+def _configure_db_create_params(
```

Comment:
@troydai / @tjprescott Not sure but just wanted to ask you guys whether we have any better way to handle this like manipulating commands based on conditions?